_config.php

```diff
@@ -80,27 +80,31 @@
 
     // Enable insert-link to internal pages
     $cmsModule = ModuleLoader::inst()->getManifest()->getModule('silverstripe/cms');
-    $cwpEditor
-        ->enablePlugins([
-            'sslinkinternal' => $cmsModule
-                ->getResource('client/dist/js/TinyMCE_sslink-internal.js'),
-            'sslinkanchor' => $cmsModule
-                ->getResource('client/dist/js/TinyMCE_sslink-anchor.js'),
-        ]);
+    if ($cmsModule) {
+        $cwpEditor
+            ->enablePlugins([
+                'sslinkinternal' => $cmsModule
+                    ->getResource('client/dist/js/TinyMCE_sslink-internal.js'),
+                'sslinkanchor' => $cmsModule
+                    ->getResource('client/dist/js/TinyMCE_sslink-anchor.js'),
+            ]);
+    }
 
     // Re-enable media dialog
     $assetAdminModule = ModuleLoader::inst()->getManifest()->getModule('silverstripe/asset-admin');
-    $cwpEditor
-        ->enablePlugins([
-            'ssmedia' => $assetAdminModule
-                ->getResource('client/dist/js/TinyMCE_ssmedia.js'),
-            'ssembed' => $assetAdminModule
-                ->getResource('client/dist/js/TinyMCE_ssembed.js'),
-            'sslinkfile' => $assetAdminModule
-                ->getResource('client/dist/js/TinyMCE_sslink-file.js'),
-        ]);
-    $cwpEditor->insertButtonsAfter('table', 'ssmedia');
-    $cwpEditor->insertButtonsAfter('ssmedia', 'ssembed');
+    if ($assetAdminModule) {
+        $cwpEditor
+            ->enablePlugins([
+                'ssmedia' => $assetAdminModule
+                    ->getResource('client/dist/js/TinyMCE_ssmedia.js'),
+                'ssembed' => $assetAdminModule
+                    ->getResource('client/dist/js/TinyMCE_ssembed.js'),
+                'sslinkfile' => $assetAdminModule
+                    ->getResource('client/dist/js/TinyMCE_sslink-file.js'),
+            ]);
+        $cwpEditor->insertButtonsAfter('table', 'ssmedia');
+        $cwpEditor->insertButtonsAfter('ssmedia', 'ssembed');
+    }
```

Comment:
@tractorcow perhaps we should move these blocks into cwp/cwp instead, which will have a recipe-cms dependency?

Reply:
Yes I think so, but you only need an `silverstripe/admin` dependency, not the whole recipe.

Reply:
Yeah, cool =)

Reply:
Ok, I've moved all of the TinyMCE configuration to cwp/cwp here: https://github.com/silverstripe/cwp/pull/36